addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -134,6 +134,12 @@ const Wysiwyg = Widget.extend({
                     if (!$field.hasClass('o_editable_date_field_format_changed')) {
                         $linkedFieldNodes.text($field.data('oe-original-with-format'));
                         $linkedFieldNodes.addClass('o_editable_date_field_format_changed');
+                        const originalDate = $field.data('oe-original');
+                        $(`.oe_hide_on_date_edit[data-oe-original='${originalDate}']`)
+                            .addClass('d-none');
+                        Wysiwyg.setRange(
```

Comment:
As discussed, this need a comment, it is not obvious IMHO why we need this, something like:
```js
// we might hide the clicked date, focus the one supposed to be editable
```

Reply:
This line doesn't do anything, you can modify it to
```js
if (window.location.href.includes('localhost')) {
                            // we might hide the clicked date, focus the one
                            // supposed to be editable
                            console.log("setting range on ");
                            console.log($linkedFieldNodes.filter(':not(.oe_hide_on_date_edit)')[0]);
                            Wysiwyg.setRange($linkedFieldNodes.filter(':not(.oe_hide_on_date_edit)')[0]);
                        }
                        ```
And try side by side on localhost & 127.0.0.1 testing all events dates field (both on /event and on an event register page), with or without this line, doesn't change anything, when the clicked date is hidden, the visible one doesn't get the cursor